.changelog/2737.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+resource/mongodbatlas_project_api_key: Fixes multiple issues
```

Comment:
any chance we can actually be more clear since this is what customers might read?

Reply:
agree, we can have separate entry for the bug fix of in project_assignments and then the change of the import id format

Reply:
updated to have more detail in changelog